common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Strategy.java

```diff
@@ -488,7 +497,7 @@ protected TokenResult getTokenResultFromHttpResponse(@NonNull final HttpResponse
         return result;
     }
 
-    private String getCloudSpecificTenantEndpoint(
+    private String buildCloudSpecificTokenEndpoint(
             @NonNull final MicrosoftStsAuthorizationResponse response) {
         if (!StringUtil.isEmpty(response.getCloudGraphHostName())) {
```

Comment:
@iambmelt I believe this should have been `if (!StringUtil.isEmpty(response.getCloudInstanceHostName()))`? as that is the one we use below. I'm going to make the change but wanted to double check if this was on purpose?

Reply:
Looking....

Reply:
This change was actually introduced in:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/commit/371212f51df69f7ad4bfafc3980b37c50ed2067b

It appears to be a typo

Reply:
Thanks, I'll make the change in this PR.